app/javascript/dashboard/store/modules/integrations.js

```diff
@@ -0,0 +1,85 @@
+/* eslint no-param-reassign: 0 */
+import * as MutationHelpers from 'shared/helpers/vuex/mutationHelpers';
+import * as types from '../mutation-types';
+import IntegrationsAPI from '../../api/integrations';
+
+const state = {
+  records: [],
+  uiFlags: {
+    isFetching: false,
+    isFetchingItem: false,
+    isUpdating: false,
+  },
+};
+
+export const getters = {
+  getIntegrations($state) {
+    return $state.records;
+  },
+  getIntegration: $state => integrationId => {
+    const [integration] = $state.records.filter(
+      record => record.id === integrationId
```

Comment:
Parse integrationId with `parseInt`

Reply:
integrationId isn't an integer. it would be of the format `slack`